Python/ceval.c

```diff
@@ -46,7 +46,7 @@
 #  error "ceval.c must be build with Py_BUILD_CORE define for best performance"
 #endif
 
-#if !defined(Py_DEBUG) && !defined(Py_TRACE_REFS)
+#if !defined(Py_DEBUG) && !defined(Py_TRACE_REFS) && !defined(Py_NOGIL)
```

Comment:
Have you checked what the performance impact of this on windows is?

Reply:
It seems roughly an extra ~10% regression on Windows pgo. I intend to implement this later, but I'd like to keep this PR small.